### DISCUSSION TITLE
[mac] Unread message counter in dock not working

### DISCUSSION THREAD

**Discussion by sennabecool** [ankurk91/google-chat-electron#92] [General]
Everything works perfectly but the message counter in the dock witch is quite inconvenient. The only way to know I have unread messages is to check the tray icon that displays a red dot when it is the case. 

Nontheless, I'm very grateful that this native solution exists, thx a lot ! 

App Version: 2.15.1 
OS Mac Monterey 12.3.1 (21E258)

<img width="34" alt="Capture d’écran 2022-05-19 à 17 03 03" src="https://user-images.githubusercontent.com/12663602/169328425-6cbd3203-1d32-456d-b80e-a04cfed82794.png">

**Comment by ankurk91** [comment]
I could not understand, what problem you are facing.

**Reply by sennabecool** [reply]
Notifications are not working on the dock icon

**Comment by ankurk91** [comment]
Do you see unread message count on the dock?

**Reply by sennabecool** [reply]
No 

I have checked in system preference and it is enabled, but not working.

**Comment by ankurk91** [comment]
I will check with my Mac VM and will let you know

**Reply by sennabecool** [reply]
Thank you !

**Comment by ankurk91** [comment]
![Screenshot from 2022-05-20 09-49-22](https://user-images.githubusercontent.com/6111524/169449826-0b09d698-949a-4781-b1cb-aa0d97621341.png)

I could not reproduce the issue on my Mac VM, (v10.15)
I can see the unread message count from individuals and groups.
